GetJobber/help_theme_2019
# Homepage/features

## What was done
The remainder of the home page styling. 
- The Learn about our Features section
- Jobber Resources
- Midpage CTA
- Highlight text 
- Button styling for homepage
- Some overriding of Zendesk color variables with our colors

## What wasn't done
- Footer styling was pulled out as there is a PR coming with the full footer style

## How to test

Features hover state
![Screenshot 2019-12-17 10 41 00](https://user-images.githubusercontent.com/1953515/71040170-3fe72780-20e3-11ea-9692-8db1cea9235f.png)

Features focus state (tabbing through links)
![Screenshot 2019-12-17 10 41 07](https://user-images.githubusercontent.com/1953515/71040177-45dd0880-20e3-11ea-8f19-4b6aff8fc557.png)

Jobber resources and mid-page CTA should look close to the sketch file.

Button in the head (chat now) should be ghost 2px thick border that inverts colours on hover.
Button in the mid page CTA should be primary that inverts on hover.
